.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 .DS_Store
+package-lock.json
```

Comment:
`package-lock.json` should not be ignored, according to https://github.com/npm/npm/blob/v5.0.0/doc/files/package-lock.json.md

Reply:
Sure, but so far it is not part of the repo and I guess you (the team) ignored it globally or do not want to add the lockfile (which woul require manual dependency upgrades then).

Reply:
We're keen to check it in from now on 👍